src/inbox/mod_inbox_rdbms.erl

```diff
@@ -61,6 +63,27 @@ get_inbox_rdbms(LUser, LServer, #{ order := Order } = Params) ->
                  " ORDER BY timestamp ", OrderSQL, ";"],
     mongoose_rdbms:sql_query(LServer, Query).
 
+
+get_inbox_unread(Username, Server) ->
+    case mongoose_rdbms:sql_query(Server,
+                                  ["select unread_count from inbox "
+                                   "WHERE luser=", esc_string(Username), 
+                                   " AND lserver=", esc_string(Server), ";"]) of
+        {selected, []} ->
+            ok;
+        {selected, [{CountBin0}]} ->
+            Count = try binary_to_integer(CountBin0) of
```

Comment:
We clearly know what to expect here, so why not `case _ is_binary(CountBin0}) -> ...; _ is_integer(CountBin0}) -> ... end` instead of an exception?